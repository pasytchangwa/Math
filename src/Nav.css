.title {
  font-size: 36px;
}

h1 {
  text-align: center;
}

nav {
  background-color: darkgoldenrod;
  padding: 2rem;
  display: flex;
  justify-content: space-between;
  align-items: baseline;
}

nav ul {
  width: 260px;
  display: flex;
  justify-content: space-between;
  font-style: italic;
  color: white;
  list-style-type: none;
  text-decoration: none;
}

nav li {
  font-size: 22px;
  background-color: darkred;
  list-style: none;
}

.middle {
  padding: 0 0.8rem;
}
