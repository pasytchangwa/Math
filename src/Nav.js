import React from 'react';
import { Link } from 'react-router-dom';
import './Nav.css';

const Nav = () => {
  const navStyle = {
   color: 'black',
  };
  const dec = {
    textDecoration: 'none',
  }
  return (
    <nav>
      <h2 className="title">Math magicians</h2>
      <ul>
        <li style={navStyle}>
          <Link style={dec} to="/home">
          Home |
          </Link>
        </li>
        <li style={navStyle}>       <Link style={dec} to="/calculate">Calculator |
        </Link>
        </li>
        <li style={navStyle}>
         <Link style={dec} to="/quote">Quote
         </Link>
        </li>
     </ul>
    </nav>
  );
}

export default Nav;
