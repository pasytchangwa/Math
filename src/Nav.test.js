import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from './Nav';

test('It should render correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Nav />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
