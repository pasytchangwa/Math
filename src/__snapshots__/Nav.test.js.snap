// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render correctly 1`] = `
<nav>
  <h2
    className="title"
  >
    Math magicians
  </h2>
  <ul>
    <li
      style={
        Object {
          "color": "black",
        }
      }
    >
      <a
        href="/home"
        onClick={[Function]}
        style={
          Object {
            "textDecoration": "none",
          }
        }
      >
        Home |
      </a>
    </li>
    <li
      style={
        Object {
          "color": "black",
        }
      }
    >
      <a
        href="/calculate"
        onClick={[Function]}
        style={
          Object {
            "textDecoration": "none",
          }
        }
      >
        Calculator |
      </a>
    </li>
    <li
      style={
        Object {
          "color": "black",
        }
      }
    >
      <a
        href="/quote"
        onClick={[Function]}
        style={
          Object {
            "textDecoration": "none",
          }
        }
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
